src/jackknife.jl

```diff
@@ -0,0 +1,45 @@
+"""
+Use bootweights to creaye replicate weights using Rao-Wu bootstrap. The function accepts a `SurveyDesign` and returns a `ReplicateDesign` which has additional columns for replicate weights. 
+
+```jldoctest
+julia> using Random
+
+julia> dclus1 = SurveyDesign(apiclus1; clusters = :dnum, popsize=:fpc);
+```
+"""
+function jknweights(design::SurveyDesign; replicates = 10, rng = MersenneTwister(1234))
+    stratified = groupby(design.data, design.strata)
+    H = length(keys(stratified))
+    substrata_dfs = DataFrame[]
+    for h = 1:H
+        substrata = DataFrame(stratified[h])

```

Comment:
if this is JKn, for stratified samples, what would JK1 be like, which is for unstratified samples? Is JK1 just equivalent to having one strata? If so, then does this function effectively work as JK1 when no strata?

Reply:
If there is no stratification, assume there is one strara and everything falls in it. This means, we don't need JK1, only JKn will do.

Reply:
Is it correct? You mean don't need to make any changes now